include/fastdds/rtps/transport/SocketTransportDescriptor.h

```diff
@@ -19,13 +19,13 @@
 
 #ifdef _WIN32
 #include <cstdint>
-#endif
+#endif // ifdef _WIN32
```

Comment:
I don't think `cstdint` this is harmful on non-win32 platforms, so I think it is safe to remove the conditionals